README.md

```diff
@@ -251,6 +251,29 @@ export TRITON_OVERRIDE_DIR=<override_dir>
 # Step 4: Run the kernel again to see the overridden result
 ```
 
+**Compiler Pipeline Inspection Steps**
+```bash
+# To introspect the pipeline add_stages, set the following environments:
+export TRITON_INSPECT_PASS_STAGES=1
+```
```

Comment:
why do we need an env var here? We can decide to call the hook or not based on whether it is None

Reply:
Ah, yeah I guess so.